ndb/src/google/cloud/ndb/model.py

```diff
@@ -1026,6 +1166,42 @@ def call(value):
 
         return call
 
+    def _apply_to_values(self, entity, function):
+        """Apply a function to the property value / values of a given entity.
+
+        This retrieves the property value, applies the function, and then
+        stores the value back. For a repeated property, the function is
+        applied separately to each of the values in the list. The
+        resulting value or list of values is both stored back in the
+        entity and returned from this method.
+
+        Args:
+            entity (Model): An entity to get a value from.
+            function (Callable[[Any], Any]): A transformation to apply to
+                the value.
+
+        Returns:
+            Any: The transformed value store on the entity for this property.
+        """
+        value = self._retrieve_value(entity, self._default)
+        if self._repeated:
+            if value is None:
+                value = []
+                self._store_value(entity, value)
+            else:
+                # NOTE: This assumes, but does not check, that ``value`` is
```

Comment:
Is this a verbatim paste from old code?

Reply:
No. I added it. No such comment existed before.